src/overlays/actors/ovl_Obj_Hana/z_obj_hana.c

```diff
@@ -27,11 +26,71 @@ const ActorInit Obj_Hana_InitVars = {
     (ActorFunc)ObjHana_Update,
     (ActorFunc)ObjHana_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 8, 10, 0, { 0, 0, 0 } },
+};
+
+// sColChkInfoInit
+static CollisionCheckInfoInit sColChkInfoInit = { 0, 0xC, 0x3C, 0xFF };
+
+typedef struct {
+    /* 0x00 */ Gfx* dList;
+    /* 0x04 */ f32 scale;
+    /* 0x08 */ f32 yOffset;
+    /* 0x0C */ s16 radius;
+    /* 0x0E */ s16 height;
+} HanaParams; // size 0x10
+
+HanaParams D_80B93AA4[] = { { 0x05000500, 0.00999999977648f, 0.0f, -1, 0 },
```

Comment:
You should be able to simplify at least some if not all of these floats. Try 0.01f, 0.1f, and 0.4f?